.github/workflows/cifuzz.yml

```diff
@@ -4,6 +4,9 @@ on:
     paths-ignore:
       - 'doc/**'
       - '**/*.md'
+    branches:
+      - main
+      - release-2.3
```

Comment:
Just `main`. I can delete `2.3` from the other CI/CD files, since that branch is now stale.

Reply:
OK